contracts/src/v0.8/dev/vrf/VRFCoordinatorV2Plus.sol

```diff
@@ -325,7 +325,7 @@ contract VRFCoordinatorV2Plus is VRF, SubscriptionAPI {
   function computeRequestId(
     bytes32 keyHash,
     address sender,
-    uint64 subId,
+    uint256 subId,
```

Comment:
Ah I think this would change the preseed for sure. That would explain the proof change.